OptimizelySDK.Tests/TestData.json

```diff
@@ -85,7 +85,8 @@
 						"id": "155561",
 						"value": "red"
 					}
-				]
+				],
+				"featureEnabled": true
```

Comment:
What's the need to add featureEnabled in rollout? It's only for variations associated with experiment. 
@oakbani FYI

Reply:
As per doc 'featureEnabled' will be part of rollout experiment but will always be true. And should not be taken into account when decision source is rollout.

Reply:
And this line I think is part of experiment.